modules/4337/docs/v0.2.0/audit.md

```diff
@@ -16,6 +18,22 @@ There is one acknowledged finding from the audit report:
   > - The most critical functionality, such as signature checks and replay protection, is handled by the Safe and Entrypoint contracts.
   > - The entrypoint contract uses the optimizer.
 
-## Files
+### Files
 
 - [Final audit report](audit-report-v2.0.pdf)
+
+## Audit 2
+
+### Auditor
+
+OpenZeppelin (<https://openzeppelin.com/>).
+
+### Notes
+
+The final audit was performed on commit [3853f34f31837e0a0aee47a4452564278f8c62ba](https://github.com/safe-global/safe-modules/tree/3853f34f31837e0a0aee47a4452564278f8c62ba).
+
+No security-related findings were reported, but we have implemented some of the suggested optimisations from the audit report into the codebase. These changes will be included in the next release of the 4337 module. For more information about the audit findings, please refer to the report.
```

Comment:
I would reword this a bit - while there were security related findings (M-01, L-01 to 03), none of them led to actual Solidity outside of comments and pragmas.

Reply:
Hmm, not sure I follow, for me security findings mean something that endager users' assets and require an immediate response. The issues you mentioned are about missing docstring, floating pragma and potential incompatibility. For me they're not really security related. Maybe we're looking for a better term.

Reply:
> security findings mean something that endanger users' assets and require an immediate response

Personally, I tend to agree, but OZ audit qualified M-01 and L-0[1-3] as "security issues" in the report.

**Edit**: Nevermind, they are just "issues". I would still reword this to say that no issues were found that led to Solidity code changes.